paparazzi/src/main/java/app/cash/paparazzi/internal/interceptors/MatrixVectorMultiplicationInterceptor.kt

```diff
@@ -19,19 +19,18 @@ object MatrixVectorMultiplicationInterceptor {
     val y = rhsVec[rhsVecOffset + 1]
     val z = rhsVec[rhsVecOffset + 2]
     val w = rhsVec[rhsVecOffset + 3]
+    val o = lhsMatOffset
     resultVec[resultVecOffset + 0] =
-      lhsMat[I(0, 0, lhsMatOffset)] * x + lhsMat[I(1, 0, lhsMatOffset)] * y + lhsMat[I(2, 0, lhsMatOffset)] * z + lhsMat[I(3, 0, lhsMatOffset)] * w
```

Comment:
revert and add `"ktlint:standard:property-naming"` to `@Suppress` above `intercept` method